ui/store/actions.js

```diff
@@ -1283,6 +1283,79 @@ export function addToken(
   };
 }
 
+export function addCollectible(address, tokenID, dontShowLoadingIndicator) {
```

Comment:
Minor - should we make this `showLoadingIndicator = true`?  I prefer arguments for _taking_ action, not _preventing_ action?  Just a discussion point, not a blocker.

Reply:
Sure I don't mind changing this. We do use it [elsewhere in this file](https://github.com/MetaMask/metamask-extension/blob/4a90e5bc84567f4a3465fcd33b2f76f79a9e1c83/ui/store/actions.js#L675). But generally I agree with the pattern you're advocating.

Reply:
It doesn't seem that we destructure arguments anywhere in these action methods... I can't think of a reason why we wouldn't, but without doing so we can't set a default value. 🤔 @darkwing